lib/pbench/test/unit/server/conftest.py

```diff
@@ -789,7 +789,12 @@ def tarball(tmp_path):
     datafile = tmp_path / filename
     metadata = ConfigParser()
     metadata.add_section("pbench")
+    metadata.add_section("run")
```

Comment:
What about the removal of `pbench-server-prep-shim-002` required changes to this test configuration?